Kitodo/src/main/webapp/WEB-INF/templates/includes/importConfigurationEdit/opacSearchFields.xhtml

```diff
@@ -0,0 +1,161 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:f="http://xmlns.jcp.org/jsf/core"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:p="http://primefaces.org/ui">
+    <f:view>
+        <!--@elvariable id="isViewMode" type="boolean"-->
+        <p:panelGrid columns="2"
+                     layout="grid">
+            <p:row>
+                <div>
+                    <p:outputLabel for="interfaceType" value="#{msgs['importConfig.field.interfaceType']} *"/>
+                    <p:selectOneMenu id="interfaceType" styleClass="input-with-button"
+                                     required="#{not empty param['editForm:save']}"
+                                     value="#{importConfigurationEditView.importConfiguration.interfaceType}">
+                        <f:selectItem itemValue="#{null}"
+                                      itemLabel="-- #{msgs['importConfig.selectInterfaceType']} --"
+                                      noSelectionOption="true"/>
+                        <f:selectItems value="#{importConfigurationEditView.interfaceTypes}"/>
+                        <p:ajax event="change"
+                                oncomplete="toggleSave()"
+                                update="editForm:importConfigurationTabView"/>
+                    </p:selectOneMenu>
+                    <p:commandButton id="interfaceTypeHelp" type="button"
+                                     styleClass="help-button" icon="fa fa-lg fa-question-circle-o"/>
+                    <p:tooltip for="interfaceTypeHelp"
+                               value="Type of the search interface"/>
```

Comment:
Please add i18n keys for all tooltip messages.

Reply:
see above